test/Feature/StructureSpec.hs

```diff
@@ -78,46 +85,53 @@ spec = do
 
           deleteResponse `shouldBe` Just "No Content"
 
-    it "includes definitions to tables" $ do
-        r <- simpleBody <$> get "/"
-
-        let def = r ^? key "definitions" . key "child_entities"
-
-        liftIO $
-
-          def `shouldBe` Just
-            [aesonQQ|
-              {
-                "type": "object",
-                "description": "child_entities comment",
-                "properties": {
-                  "id": {
-                    "description": "child_entities id comment\n\nNote:\nThis is a Primary Key.<pk/>",
-                    "format": "integer",
-                    "type": "integer"
-                  },
-                  "name": {
-                    "description": "child_entities name comment",
-                    "format": "text",
-                    "type": "string"
-                  },
-                  "parent_id": {
-                    "description": "Note:\nThis is a Foreign Key to `entities.id`.<fk table='entities' column='id'/>",
-                    "format": "integer",
-                    "type": "integer"
+      it "includes definitions to tables" $ do
+          r <- simpleBody <$> get "/"
```

Comment:
As long as your changing indentation, these lines should be indented 2 beneath the `it` statement rather than 4.